qiskit/extensions/standard/crz.py

```diff
@@ -15,19 +15,19 @@
 """
 controlled-rz gate.
 """
-from qiskit.circuit import Gate
+from qiskit.circuit import ControlledGate
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit import QuantumRegister
 from qiskit.extensions.standard.u1 import U1Gate
 from qiskit.extensions.standard.cx import CnotGate
 
 
-class CrzGate(Gate):
+class CrzGate(ControlledGate):
     """controlled-rz gate."""
 
     def __init__(self, theta):
         """Create new crz gate."""
-        super().__init__("crz", 2, [theta])
+        super().__init__("crz", 2, [theta], num_ctrl_qubits=1)
 
```

Comment:
add `self.base_gate` here for the crz gate?

Reply:
thanks